src/Entity/ParagraphsRowType.php

```diff
@@ -30,6 +30,12 @@
  *     "label" = "label",
  *     "uuid" = "uuid"
  *   },
+ *   config_export = {
+ *     "id",
+ *     "label",
+ *     "description",
+ *     "behavior_plugins",
+ *   },
```

Comment:
Huh, this is good to know and something that should be brought up in dev chat so that others also are aware. Please add an item to the next agenda.

Reply:
I added it to our Dec 10th agenda